reference/Playback-API.yaml

```diff
@@ -0,0 +1,160 @@
+tags:
+  - name: Playback API
+servers:
+  - url: "https://playback-api-development.dev.streamamg.com"
+paths:
+  /playback:
+    get:
+      summary: Get playback data
+      operationId: get-playback-data
+      parameters:
+        - name: entryId
+          in: query
+          schema:
+            type: string
+          required: true
+          description: "Entry ID in MP"
+        - name: accessToken
+          in: query
+          schema:
+            type: string
+          required: false
+          description: "User access token"
+        - name: clientId
+          in: query
+          schema:
+            type: string
+            format: uuid
+          required: true
+          description: "Client ID retrieved from CloudPay"
+      responses:
+        "200":
+          description: "Success - get playback data"
+          content:
+            application/json:
+              schema:
+                  $ref: "#/components/schemas/PlaybackEndpointResponse"
+              examples:
+                Example - Success VoD:
+                  value:
+                    id: "0_4l1aj0bo"
+                    name: "World Cup of Tests"
+                    description: "Test Video"
+                    thumbnail: "https://video-store/test/12345"
+                    duration: "2800000"
+                    media:
+                      {
+                        hls: "https://open.http.mp.streamamg.com/p/Test-Partner/sp/Test-Partner00/playManifest/entryId/${entryId}/flavorIds/0_4l1aj0bo,0_522aj0tt/format/applehttp/protocol/https/a.m3u8?ks=YTlhYzYzNmJmOWVhMDQzNzRjYTY3M",
+                      }
+                Example - Success Live:
+                  value:
+                    id: "0_4l1aj0bo"
+                    name: "World Cup of Tests"
+                    description: "Test Video"
+                    thumbnail: "https://video-store/test/12345"
+                    duration: "2800000"
+                    media:
+                      {
+                        mpegdash: "url-to-mpegdash-manifest",
+                        hls: "url-to-hls-manifest",
+                        applehttp: "url-to-applehttp-manifest",,
+                      }
+        "400":
+          description: "Bad Request"
+          content:
+            application/json:
+              schema:
+                description: "Error getting session identifier"
+                type: object
+                properties:
+                  status:
+                    type: number
+                    enum:
+                      - 400
+                  message:
+                    type: string
+                    minLength: 1
+                required:
+                  - status
+                  - message
+              examples:
+                Invalid Request:
+                  value:
+                    status: 400
+                    message: Bad Request Exception
+        "500":
```

Comment:
The 500 response example has `status` and `message`, and schema has denoted `message` only, can you please fix it?

Reply:
You're right, I add status as well.